### DISCUSSION TITLE
Yandex SMTP not Working

### DISCUSSION THREAD

**Discussion by adrianpaniagualeon** [alfio-event/alf.io#1288] [Q&A]
Hi, 
I have configured the SMTP settings like this
SMTP hostname: smtp.yandex.com
SMTP port: 465
SMTP Protocol (smtp or smtps): smtp
SMTP Username: myemail@mydomain.com
(The username and password are correct)

But the emails stay WAITING in the queue. 
In the log said Mail Server Connection Failed: Connection Reset

**Comment by cbellone** [comment]
Hi @adrianpaniagualeon 
please double check that the event has not expired.
Alf.io does not send (yet) emails for expired events.
